assets/sass/components/_footer.scss

```diff
@@ -0,0 +1,163 @@
+.footer {
+  width: 100%;
+&__wrapper {
+  margin: 0;
+  padding: 0;
+  border:0;
+  background-color: #333333;
+  width: 100%;
+ }
+&__adress {
+  padding-bottom: 50px;
+  margin:0;
+  padding-top: 100px;
+  font-size: 14px;
+    width: 100%;
+}
+&__adress p {
+  padding-bottom: 10px;
+}
+&__adress p:first-child{
+  font-size: 16px;
+  padding-bottom: 22px;
+}
+&__adress p:last-child{
+  padding-bottom: 0;
+}
+p {
+  margin-left: 38px;
+  color: #FFFFFF;
+  font-family: Montserrat, SemiBold;
+  font-size: 16px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+&__email{
+  padding-bottom: 50px;
+}
+&__email p:first-child{
```

Comment:
Same as address class selector. No needed to write &__email multiple times, when you wrote it once that is it. Please read about nesting one more time. https://sass-lang.com/guide -> Nesting part